### PR TITLE
Enable cloud benchmarks from k8s cluster

### PR DESCRIPTION
**Pull request by menski**
## Description

- add `noop` process for baseline testing
- add configuration option to enable tls for clients
- allow to set broker url by `ZEEBE_ADDRESS` env variable as used in cloud env
- update Dockerfile to use java 11

### CODE REVIEW

File: benchmarks/project/src/main/resources/application.conf [hunk 1]
```diff
@@ -1,5 +1,7 @@
 app {
   brokerUrl = "localhost:26500"
+  brokerUrl = ${?ZEEBE_ADDRESS}
```
Comment (Zelldon) [inline benchmarks/project/src/main/resources/application.conf]: How does this work with the line above? Is this only evaluated and assigned  when it exists?

**Review by ChrisKujawa — APPROVED** [review]
Looks good! Just one small question

File: benchmarks/project/src/main/resources/application.conf (hunk 1 above)
Comment (menski) [reply benchmarks/project/src/main/resources/application.conf]: Yes that's I guess the meaning of the `?`, so if `ZEEBE_ADDRESS` does not exists it's ignored

**Pull request merged**